VirtualH89/Src/computer.h

```diff
@@ -10,8 +10,8 @@
 #define COMPUTER_H_
 
 #include "h89Types.h"
+#include "AddressBus.h"
```

Comment:
This helps avoid the header include graph from becoming a tangled mess. So if the header only uses a pointer or reference to the class, then a forward declare works. If the class is actually used then including the other header is fine. So don't rework any code, but if they can be forward declared, I'd prefer it.